### ISSUE TITLE
Upgrade from 1.4.7 to 1.5.0 appears to corrupt curves

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#2552]
I've seen this both on my tx and the simulator. 

Please see the following model definition which I've copied from a 1.4.7 backup.

[2a2f v-tail v3.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/11077770/2a2f.v-tail.v3.zip)

If you now upload that to 1.5.0 and look at the curves you will see that many of them are corrupt. 

e.g. 

this is curve 1 on 1.4.7 

![image](https://user-images.githubusercontent.com/96293240/227922565-813d13fe-7255-48a6-b1b9-4455fcc1c863.png)

and on 1.5.0

![image](https://user-images.githubusercontent.com/96293240/227921784-78d0f32e-a7cc-4e14-afec-919cfcf31c3d.png)

similarly for curve 10 on 1.4.7

![image](https://user-images.githubusercontent.com/96293240/227922706-ce4df115-cd3c-4e5b-8942-a87c1a603d7e.png)

and on 1.5.0 

![image](https://user-images.githubusercontent.com/96293240/227921961-806f5f54-ecec-4521-a1da-3a85b9245256.png)

these are just a couple of examples, I think most of the curves are actually corrupt.

Thanks

**Comment by mawzthefinn** [comment]
Which 1.5.0 nightly are you using for this test?

**Comment by BladeScraper-Designs** [comment]
I am able to confirm this issue, I just upgraded to 1.5.0 nightly on my X20S to test it.  I can't say which *specific* nightly, but it's whatever version Ethos Suite does when you choose Nightly -> nightly as of 7:40AM or so CDT today.

I initially expected to come on here and say "I just upgraded mine and none of the curves were affected", as the model I checked and screenshotted prior to the upgrade was unaffected by the upgrade.  However, I wewnt to another model and checked that, and yes, some of my curves were messed up during the upgrade.  
![screenshot-2023-03-26-36726](https://user-images.githubusercontent.com/40482965/227944362-196d1945-92e0-450f-891b-c27ee205a3b0.jpg)
![screenshot-2023-03-26-36740](https://user-images.githubusercontent.com/40482965/227944373-ef03e2bc-d836-4257-a5a6-124b8d31ca64.jpg)
Some, however, were unaffected.
![screenshot-2023-03-26-36747](https://user-images.githubusercontent.com/40482965/227944390-64da1fb9-4088-4aa6-bd10-6c2c2a5ecc43.jpg)
